### DISCUSSION TITLE
Mentioned your project

### DISCUSSION THREAD

**Discussion by MichaelCurrin** [Ileriayo/markdown-badges#314] [General]
Hi there

I found your project in my GH feed

I know of a few other badge projects. I added yours to my list:

https://github.com/MichaelCurrin/badge-generator/blob/master/docs/related-projects.md

And also on this page of my app:  https://michaelcurrin.github.io/badge-generator/#/catalogue

My  catalogue is small, though it includes links.

**Comment by MichaelCurrin** [comment]
You might also like to try out my Generic badge generator to make badges for you to add to your catalogue.

https://michaelcurrin.github.io/badge-generator/#/generic

My form lets you put in size, color, logo, text, etc. and gives you a neat Markdown snippet

**Reply by Ileriayo** [reply]
Hey @MichaelCurrin, you've got a great project there.

Thanks for sharing!

**Comment by sharynneazhar** [comment]
@MichaelCurrin @Ileriayo I created this generator based on the badges already created in this repo. I got lazy with copy-pasta. Thought I'd share! 🙂 

https://github.com/sharynneazhar/markdown-badges-generator

**Reply by Ileriayo** [reply]
This is fantastic @sharynneazhar!
Great job 👏

**Reply by kohasummons** [reply]
Heyy Sharynne, just took your generator for a spin and I loveeee it. Thanks for contributing to the community. 🔥 

@sharynneazhar
